libgit2 required for CI due to changes in usethis

I have a [recently failing CI check](https://github.com/cmu-delphi/covidcast/pull/366/checks?check_run_id=1531674152) that seems to be triggered by an update to `usethis`. The [usethis changelog](https://usethis.r-lib.org/news/index.html) says v2.0.0 is adopting `gert`. [Documentation for gert](https://cran.rstudio.com/web/packages/gert/index.html) says, "On Linux, a somewhat recent version of 'libgit2' is required"

The CI log shows `usethis` updating to v2.0.0, `gert` being installed and a failure to find `libgit2` followed by erroring out. The relevant portions of the log are:

```
gert        (NA    -> 1.0.2) [CRAN]
usethis     (1.6.3 -> 2.0.0) [CRAN]
...
* installing *source* package ‘gert’ ...
** package ‘gert’ successfully unpacked and MD5 sums checked
** using staged installation
Using PKG_CFLAGS=
Using PKG_LIBS=-lgit2
----------------------------- ANTICONF -------------------------------
Configuration failed to find libgit2 library. Try installing:
 * brew: libgit2 (MacOS)
 * deb: libgit2-dev (Debian, Ubuntu, etc)
 * rpm: libgit2-devel (Fedora, CentOS, RHEL)
If libgit2 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a libgit2.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
ERROR: configuration failed for package ‘gert’
-------------------------- [ERROR MESSAGE] ---------------------------
* removing ‘/home/runner/work/_temp/Library/gert’
<stdin>:1:10: fatal error: git2.h: No such file or directory
compilation terminated.
----------------------------------------------------------------------
Error in i.p(...) : 
  (converted from warning) installation of package ‘gert’ had non-zero exit status
Calls: <Anonymous> ... with_rprofile_user -> with_envvar -> force -> force -> i.p
Execution halted
Error: Process completed with exit code 1.
```

--- a/.github/workflows/r_covidcast_ci.yml
+++ b/.github/workflows/r_covidcast_ci.yml
@@ -33,7 +33,7 @@ jobs:
       - name: Install linux dependencies
         run: |
             sudo apt-get install libcurl4-openssl-dev
-            sudo apt-get install libgit2
+            sudo apt-get install libgit2-dev
       - name: Cache R packages
         uses: actions/cache@v2
         with:

--- a/.github/workflows/r_covidcast_ci.yml
+++ b/.github/workflows/r_covidcast_ci.yml
@@ -30,8 +30,10 @@ jobs:
         uses: r-lib/actions/setup-r@v1
         with:
           r-version: ${{ matrix.r-version }}
-      - name: Install libcurl
-        run: sudo apt-get install libcurl4-openssl-dev
+      - name: Install linux dependencies
+        run: |
+            sudo apt-get install libcurl4-openssl-dev
+            sudo apt-get install libgit2
       - name: Cache R packages
         uses: actions/cache@v2
         with:

